project/Http4sPlugin.scala

```diff
@@ -75,7 +75,7 @@ object Http4sPlugin extends AutoPlugin {
     http4sMimaVersion := {
       version.value match {
         case VersionNumber(Seq(major, minor, patch), _, _) if patch.toInt > 0 =>
-          Some(s"${major}.${minor}.0")
+          Some(s"${major}.${minor}.${patch.toInt - 1}")
```

Comment:
1. Scala 2.11 is stricter than 2.12, which we didn't discover until we ran on all branches.
2. What's done is done.

Reply:
Not sure I follow on this one. We're only checking version to version not from initial binary for the minor?

Reply:
Compatibility should be transitive.  If we haven't broken compatibility from .0, these should be equivalent.  If we have broken compatibility from .0, intentionally or not, then we have to choose between going back to .0 (another breaking change) or moving on with `${patch - 1}`.